app/src/main/java/com/fall2022_group20/dataAccessLayer/Lesson/LessonSchema.java

```diff
@@ -0,0 +1,40 @@
+package com.fall2022_group20.dataAccessLayer.Lesson;
+
+import io.realm.RealmList;
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+public class LessonSchema extends RealmObject{
+    @PrimaryKey
+    private Integer id;
+
+    //took this from the diagram, how are points and score different?
```

Comment:
Points and score are the same thing, let's just go with points for the Lessons since you have that for the scenario game.

Reply:
Completed new push with update